server/selection_test.go

```diff
@@ -10,10 +10,54 @@ import (
 	"testing"
 
 	ethcommon "github.com/ethereum/go-ethereum/common"
+	"github.com/livepeer/go-livepeer/common"
 	"github.com/livepeer/go-livepeer/net"
 	"github.com/stretchr/testify/assert"
 )
 
+type stubOrchestratorStore struct {
```

Comment:
There already is a `stubOrchestratorStore` in `eth/watchers/stub.go` we _could_  move this into common and use the stub from there. Not saying this is the best option necessarily but it would avoid some code duplication perhaps.

Reply:
I think moving the stub into the `common` package could make sense since any package that is using the `OrchestratorStore` interface will already be importing the `common` package.

However, many of the tests in the `watchers` package currently contain assertions that directly access the internal state of `stubOrchestratorStore`. So, if we want to use a new `common.StubOrchestratorStore` type instead we would need to refactor the assertions to use a new public API method for checking the store's state. The public API of the stub would further increase in order to support stubbing errors (i.e. `SetSelectErr()` or `SetUpdateErr()`). A better approach that does not require adding custom public API methods could be to define `common.MockOrchestratorStore` that embeds `mock.Mock` from the `testify/mock` package.

I'm personally ok with having a small amount of duplication in this PR for now (we're only at a 2x duplication right now - more than 2 is when I generally think its worth being a bit more wary). The duplication is cheap. If *another* use case for the `common.OrchestratorStore` interface arises in another area of the codebase that requires a stub/mock *then* we can consider removing the duplication and importing a shared stub/mock from the `common` package. What do you think?

Reply:
Sounds good to me !